ciao-controller/api/api.go

```diff
@@ -96,9 +98,18 @@ func errorResponse(err error) Response {
 type Handler struct {
 	*Context
 	Handler func(*Context, http.ResponseWriter, *http.Request) (Response, error)
+	Priv    bool
```

Comment:
Very minor but i'd rather this was named something like Privileged/AdminOnly or something like that. Priv is a teeny bit vague.